tests/openbis_functions/test_openbis_interbis.py

```diff
@@ -29,12 +32,24 @@ class Constants(Enum):
     # db_url: str = "https://openbis.matolab.org/openbis/"
     db_url: str = "https://localhost:8443/openbis/"
     testing_sample_name: str = 'TESTING_SAMPLE_NAME_PYTEST_DO_NOT_DELETE'
+    testing_sample_identifier: str = "/DEFAULT/TEST_PROJECT/TESTING_SAMPLE_NAME_PYTEST_DO_NOT_DELETE"
```

Comment:
Should we use such a weird long name here? Wouldn't it be enough to use something like "PYTEST_SAMPLE". Using the git action nobody will see the sample on the webview ...

Reply:
done